bitnami/influxdb/templates/influxdb/statefulset.yaml

```diff
@@ -0,0 +1,273 @@
+{{- if eq .Values.architecture "high-availability" }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "influxdb.fullname" . }}
+  labels: {{- include "influxdb.labels" . | nindent 4 }}
+    app.kubernetes.io/component: influxdb
+spec:
+  serviceName: {{ include "influxdb.fullname" . }}-headless
+  replicas: {{ .Values.influxdb.replicaCount }}
+  updateStrategy:
+    type: {{ .Values.influxdb.updateStrategy }}
+    {{- if (eq "Recreate" .Values.influxdb.updateStrategy) }}
```

Comment:
I don't think `Recreate` is a valid `updateStrategy` for `StatefulSets`: https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#updating-statefulsets